src/pages/LoginRegister/LoginPage.css

```diff
@@ -0,0 +1,274 @@
+
```

Comment:
Please replace all styling that targets elements directly to target a className instead. This is because the styles will affect all components in the whole application if you target elements directly. 

Also, please append the page name in all of your classNames. For example, for the login page the className .form-container will be login-form-container.

Please apply this to all css and react components

Reply:
okay, I'll change that. Thank you for you suggestions.